e2e-tests/users/run

```diff
@@ -40,21 +44,42 @@ compare_mysql_cmd_local "select-5" "SHOW DATABASES;" "-h localhost -uclusterchec
 
 desc 'test monitor'
 patch_secret "my-cluster-secrets" "monitor" "$newpassencrypted"
-sleep 15
+sleep 30
 wait_cluster_consistency "$cluster" 3 1
 compare_mysql_cmd "select-4" "SHOW TABLES;" "-h $cluster-proxysql -umonitor -p$newpass"
 
 desc 'test operator'
 patch_secret "my-cluster-secrets" "operator" "$newpassencrypted"
-sleep 15
+sleep 30
+wait_cluster_consistency "$cluster" 3 1
+compare_mysql_cmd "select-4" "SHOW TABLES;" "-h $cluster-proxysql -uoperator -p$newpass"
+
+desc 'change secret name'
+kubectl_bin patch pxc $cluster --type merge --patch "$(cat $test_dir/conf/some-name.yml)"
```

Comment:
please keep it simple
```suggestion
kubectl_bin patch pxc $cluster --type merge --patch '{"spec": {"secretsName":"my-cluster-secrets-2"}}'
```

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/495/commits/3abf20cd44ee71e8dfd1d24da159625054153a18